### PR TITLE
Draggable Slider: Added expressions for more properties and some actions

### PR DESCRIPTION
**Pull request by tristanbob**
Expressions:
- Length of track
- Thickness of track
- Height of thumb

Actions:
- Change opacity of halo when pressed
- Change opacity of halo when hovered

### CODE REVIEW

File: extensions/reviewed/DraggableSliderControl.json [hunk 1]
```diff
@@ -2320,6 +2320,150 @@
           ],
           "objectGroups": []
         },
+        {
+          "description": "Thickness of track.",
```
Comment (D8H) [inline extensions/reviewed/DraggableSliderControl.json:2324]: Descriptions must contains a verb.

File: extensions/reviewed/DraggableSliderControl.json [hunk 2]
```diff
@@ -3046,6 +3190,148 @@
           ],
           "objectGroups": []
         },
+        {
+          "description": "Change opacity of halo when hovered.",
```
Comment (D8H) [inline extensions/reviewed/DraggableSliderControl.json:3194]:
```suggestion
          "description": "Change the halo opacity when the thumb is hovered.",
```

File: extensions/reviewed/DraggableSliderControl.json [hunk 3]
```diff
@@ -3046,6 +3190,148 @@
...
+          "fullName": "Halo opacity (hover)",
+          "functionType": "Action",
+          "group": "Slider thumb configuration",
+          "name": "SetHaloOpacityHover",
+          "private": false,
+          "sentence": "Change halo opacity when hovered of _PARAM0_ to _PARAM2_ px",
```
Comment (D8H) [inline extensions/reviewed/DraggableSliderControl.json:3200]:
```suggestion
          "sentence": "Change the halo opacity when hovered of _PARAM0_ to _PARAM2_ px",
```

**Review by D8H — APPROVED** [review]
There are still missing articles in sentences of this extension, but it's fine for this update.

**Comment by tristanbob** [comment]
> There are still missing articles in sentences of this extension, but it's fine for this update.

Thanks!  

What is our policy for using article words (the, a, an)? Sometimes I feel like we want things as short as possible (function title), but perhaps we want to add articles everywhere else? (short description, event sentence, etc)

**Comment by D8H** [comment]
> What is our policy for using article words (the, a, an)? Sometimes I feel like we want things as short as possible (function title), but perhaps we want to add articles everywhere else? (short description, event sentence, etc)

Sentences and description should feel as natural as possible so articles are important if English grammar requires them.
For full names, I'm not sure, I see both in built-in events, but I guess it's better to add them.
There are probably cases where articles are optional in english, but I can't really tell. What you feel is good English should be fine.

**Pull request merged**